paddle/framework/op_registry.h

```diff
@@ -400,10 +389,12 @@ class OpKernelRegistrar : public Registrar {
 /**
  * Macro to Forbid user register Gradient Operator.
  */
+/*
 #define NO_GRADIENT(op_type)                           \
-  STATIC_ASSERT_GLOBAL_NAMESPACE(                      \
-      __reg_gradient_op__##op_type##_##op_type##_grad, \
-      "NO_GRADIENT must be called in global namespace")
+ STATIC_ASSERT_GLOBAL_NAMESPACE(                      \
```

Comment:
we already have a unified registrar `REGISTER_OP_WITHOUT_GRADIENT`. I think we does not need this one any more.